.github/workflows/build.yml

```diff
@@ -85,27 +117,28 @@ jobs:
         run: |
           ../sage -python -m pip install coverage pytest-xdist
           ../sage -python -m coverage run -m pytest -c tox.ini --doctest-modules || true
-        working-directory: ./src
+        working-directory: ./worktree-image/src
         env:
           # Increase the length of the lines in the "short summary"
           COLUMNS: 120
 
-      - name: Test
-        if: always() && steps.build.outcome == 'success'
+      - name: Test --all --long
```

Comment:
```suggestion
      - name: Run all tests
```

Reply:
How about `Test all files (--all --long)`?

Reply:
I've committed something along these lines in 908ad71a14